symfony/framework-bundle/3.3/src/Kernel.php

```diff
@@ -35,6 +36,27 @@ public function registerBundles()
         }
     }
 
+    public static function bootstrapEnvironment(string $env = null)
+    {
+        $env = $env ?? $_SERVER['APP_ENV'] ?? null;
+        $projectDir = __DIR__.'/..';
+        if (!$env && !class_exists(Dotenv::class)) {
+            throw new \RuntimeException('Environment not defined. You need to define environment variables for configuration or add "symfony/dotenv" as a Composer dependency to load variables from a .env file.');
+        }
+
+        // Dotenv is not installed or generic env file is not present
+        // Assume envs are already defined
+        if (!class_exists(Dotenv::class) || !file_exists($projectDir.'/.env')) {
+            return;
+        }
+
+        $envFiles = ["$projectDir/.env"];
+        if (file_exists($file = "$projectDir/.env.$env")) {
```

Comment:
`$env` can be null here, under windows `file_exists('.env.')` will return `true`. So, if `null === $env`, then `$envFiles` will contain two elements: with `.env` and with `.env.`